src/Widgets/calendar/WeekLabels.vala

```diff
@@ -29,9 +29,10 @@ public class DateTime.Widgets.WeekLabels : Gtk.Revealer {
         vexpand = true;
 
         day_grid = new Gtk.Grid ();
+        day_grid.row_homogeneous = true;
         set_nr_of_weeks (5);
         day_grid.insert_row (1);
-        day_grid.attach (new Gtk.Separator (Gtk.Orientation.VERTICAL), 1, 0, 1, 6);
+        day_grid.attach (new Gtk.Separator (Gtk.Orientation.VERTICAL), 1, 0, 1, nr_of_weeks);
```

Comment:
Looks like this caused a regression. Best to leave it as is:

![Screenshot from 2019-07-08 13 45 39@2x](https://user-images.githubusercontent.com/7277719/60841466-c1c34400-a186-11e9-9365-d8b4200cf8fb.png)

```suggestion
        day_grid.attach (new Gtk.Separator (Gtk.Orientation.VERTICAL), 1, 0, 1, 6);
```